openshift/.pipeline/Jenkinsfile

```diff
@@ -10,14 +10,15 @@ def RABBITMQ = "rabbitmq"
 def SEARCH_API = "searchapi"
 def SAMPLE_ADAPTER = "sample-adapter"
 def ICBC_REST_ADAPTER = "icbc-rest-adapter"
+def ICBC_WEB_ADAPTER = "icbc-web-adapter"
 def DYNAMICS_ADAPTER = "dynadapter"
 def JAEGER = "jaeger"
 
 def DOMAIN = "pathfinder.gov.bc.ca"
 def SUBDOMAIN = "fams3"
 
 def GIT_URI = "https://github.com/bcgov/fams3.git"
-def GIT_REF = "master"
+def GIT_REF = "feature/icbc-web-adapter-deploy"
```

Comment:
do you want to be master ?

Reply:
yup I'll flip it after we've got the sign off and you've tested the web adapter. this is how tests the deployments.

Reply:
changed back to master